### ISSUE TITLE
Wind Waker Got Dispelled By Vacuum

### ISSUE THREAD

**Issue by owen-kalyanda** [ValveSoftware/Dota2-Gameplay#10809]
### Item name

Wind Waker

### Description

Vacuum could dispell and pull hero while wind waker is active.
Tried it on eul's scepter, it doesn't work.

### Example Match ID and Timestamp

7291592881 - 22:48

**Comment by dirubens21** [comment]
#2081

**Comment by owen-kalyanda** [comment]
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/26534357/537be976-336f-4fdd-a521-c754bd768f10)

**Comment by TheZett** [comment]
This is not a dispel, but a case of forced movement being overridden by another source of forced movement.

Since Vacuum pierces invulnerability, it can override the Cyclone's forced movement and thus cancel it, as it gets overridden by Vacuum's own forced movement.

Nether Swap can do the same, as it also pierces invulnerability.

**Comment by YashSwaraj** [comment]
> This is not a dispel, but a case of forced movement being overridden by another source of forced movement.
> 
> Since Vacuum pierces invulnerability, it can override the Cyclone's forced movement and thus cancel it, as it gets overridden by Vacuum's own forced movement.
> 
> Nether Swap can do the same, as it also pierces invulnerability. Either way, this is intended behaviour.

Shouldn't it be consistent across Eul's and Wind Waker though?

**Comment by owen-kalyanda** [comment]
so eul's should be fixed?

**Comment by TheZett** [comment]
> so eul's should be fixed?

Yes, it should work on Euls as well. It working on Windwaker is the intended behaviour, the Euls behaviour is the bug.
